main.go

```diff
@@ -38,6 +38,11 @@ func main() {
 
 	flag.Parse()
 
+	if flag.NArg() == 0 {
+		flag.Usage()
+		return
+	}
```

Comment:
The CLI doesn't accept a parameter, so this will always trigger and make it unusable

Reply:
Nuked